### ISSUE TITLE
Do not autocomplete "Gatvė." for Lithuania

### ISSUE THREAD

**Issue by tomass** [westnordost/StreetComplete#1531]
When filling a missing name of a street in Lithuania, after "g." is entered, StreetComplete automatically converts it to "Gatvė" with no apparent way to cancel this automatic change.

As per local agreement street names are entered with "g." rather than with full name "gatvė" in Lithuania. Therefore it is currently impossible to use StreetComplete to fill in missing names in Lithuania.

**Comment by westnordost** [comment]
IIRC there is a global agreement to always write out street names in full. Local communities can not override global tagging guidelines.

**Comment by westnordost** [comment]
In any case, if you want this changed in StreetComplete,  you'd have to pitch a discussion  on the tagging or talk mailing list (or other place?) and if the outcome should be that indeed local communities may override global tagging guidelines, the "don't put abbreviations in names" is actually not a global tagging guideline or the former does not apply but an exception should be made for Lithuania, then I will change it in the app.
In other words, what I want to say is that this discussion needs to happen not in this ticket but in the community.

**Comment by tomass** [comment]
Local community is the only and final decision maker. Global guidelines are just **guidelines** to increase probability that tagging is as similar as possible where possible.
Say highway classification has a description in wiki, but actual usage is different in most countries and hierarchy while essential in cartography is almost non-existent in most places.
Unfortunately it is practically impossible to agree on any standard in tagging list.
Having full word "gatvė" (even with correct lowercase) makes no IT sense in Lithuania (because of validation, integration etc.).
If it is hard to fix this in StreetComplete it's fine, looks like people are reluctant to add names with unnatural "Gatvė" via StreetComplete anyway, the rest can be fixed automatically.

**Comment by westnordost** [comment]
For highway classification, it is a matter of course that what constitutes a primary, secondary, etc. highway is different in each country, because each country has a different classification system. It suggests itself that the OSM tagging leans itself to the official classification.
But there **is** a global tagging guideline in place here, and that is that only certain values must be used for the `highway` key, and that they should be ordered in importance by `trunk`, `primary`, `secondary `etc. with special values like `living_street` and `motorway` carrying additionally special meanings. If there would be no tagging guideline, there might be `highway=zone_residentielle` in France, `highway=verkehrsberuhigter_bereich` in Germany etc.
I see StreetComplete an important factor to harmonize the taggings worldwide, making the data more usable internationally in the process.

To change the expanding behavior in the road name quest, I do not require everyone on the mailing list to agree, I require arguments brought forth on a global level that stand unrefuted. Everyone can say his opinion on the list, but what counts at the end is the facts. 

In any case, if you see no chance to bring this topic up on a global level and reach an agreement that is in your favour, I see the following additional options:
1. You can create a fork of the app that has the behavior as you wish it to be and distribute it amongst your friends and co-contributors in Lithuania. This is really easy to do, literally you just need to delete one line of code.
2. **If** it is the community consensus in Lithuania to always abbreviate street names and the community deems StreetComplete's road name quest more damaging than it being a help, I can offer to disable the quest type for Lithuania

By the way, what does "pr." stand for?

**Comment by tomass** [comment]
For `highway`. Usage and hierarchy of `unclassified/residential`(and `living_street`) is very different per country/region. The same and even worse with `track` and `service`. I do not see a chance for these to be global.

No need for fork. StreetComplete is anyway a very very good and useful app, I'm using it myself a lot and suggesting other people to use it with instructions on which campaigns are useful and which ones are not.

I lost trust in tagging list. All questions/discussions divert to something completely different extremely fast. There should be some kind of expert judges with IT, GIS and Cartography experience to have a say on important decisions. Unfortunately we have simple coders (who rarely have any IT experience) doing the most visible talking and influencing via unfortunate products like iD.

"pr." stands for "prospektas" - avenue.

**Comment by westnordost** [comment]
I know this sentiment from other people and I understand it. But we must collaborate internationally _somehow_. It cannot be a solution to withdraw in local communities and do one own's thing in terms of tagging. This way, the data will end up not being internationally comparable and the value of the global GIS database that is OSM diminished somewhat.

Interestingly, I know the most radical version of this sentiment from the developers of iD, which voiced their disregard for the global places for tagging that we currently have (mailing list, forum, wiki) several times while having from their point of view the best intentions, and best ideas about how tagging should be.
The call for a circle of experts that should dictate tagging guidelines seems convenient, but what if they then decide something that is absolutely the wrong thing in your eyes? (For example, if they decide that all names must not be abbreviated) Wouldn't you then be even more discontent?

**Comment by tomass** [comment]
I totally agree that we should try to be as global as possible, but if it contradicts a basic IT logic - there is no other way. 99% of streets signs have "g." or "g-vė". 100% official datasets (as there is one main database for streets) has "g.". So doing integration/validation with anything else would require additional logic for no benefit for local people. And we do not want full name "gatvė" on the map because it's longer. (And those, who really need full name for say text2speech have `full_name` created automatically or dynamically).

IT experts make decisions based on a lot of different **well defined** criterias. It is very clear why was one or another decision made. Impact of change is evaluated. So there is much less discontent. For example nobody with at least a fraction of IT expertise would have thought of introducing a new water tagging scheme just because "it looks nicer to some people" if water is already used in 99% of products.
